### ISSUE TITLE
A problem about paddle's width and height value in pong tutorial

### ISSUE THREAD

**Issue by celluloce** [amethyst/amethyst#1270]
In this page https://www.amethyst.rs/book/latest/pong-tutorial/pong-tutorial-02.html,
paddle's initial size is `width: 1.0, height: 1.0` but effective size is `width: 4.0, height: 16.0`.
In example code https://github.com/amethyst/amethyst/blob/master/examples/pong/pong.rs ,
those values is rewritten in `.with()` as follow,
```
        .with(Paddle {
            side: Side::Right,
            width: PADDLE_WIDTH,
            height: PADDLE_HEIGHT,
            velocity: PADDLE_VELOCITY,
        })
```
So that, I think the tutorial's page must be rewriten.

Thanks in advance.

**Comment by Toqozz** [comment]
I noticed this too.  I was going to update the book but there's a few issues which I'm not quite sure the best way to move forward on:

- `systems/paddle.rs` and `pong.rs` use `PADDLE_HEIGHT/PADDLE_WIDTH`, but `systems/bounce.rs` uses `paddle.height/paddle.width` (which is 1.0 in the current book and therefore wrong).
  - Personally I think it would be best to use the local variables wherever possible.
- `initialise_paddles()` in the examples uses the code above to correct errors, but it probably makes more sense to do this in the `Paddle` constructor.  This requires moving the `PADDLE_WIDTH/PADDLE_HEIGHT` declarations up though.

**Comment by azriel91** [comment]
Yeaps thanks guys, this was fixed in #1321.
